sys/zuse.hoon

```diff
@@ -564,10 +577,9 @@
         olz/(map flap cape)                             ::  packets completed
         wab/(map ship bath)                             ::  relationship
     ==                                                  ::
-  ++  deyd  {p/@ q/step r/?}                            ::  sig stage fake?
+  +$  deed  [=life =pass oath=(unit oath:pki:jael)]     ::  stage/pub/sig
```

Comment:
"stage" for `life` is confusing here, I haven't seen that word used for it anywhere else.